app/models/study_file.rb

```diff
@@ -24,7 +24,7 @@ class StudyFile
   # constants, used for statuses and file types
   STUDY_FILE_TYPES = ['Cluster', 'Coordinate Labels' ,'Expression Matrix', 'MM Coordinate Matrix', '10X Genes File',
                       '10X Barcodes File', 'Gene List', 'Metadata', 'Fastq', 'BAM', 'BAM Index', 'Documentation',
-                      'Other', 'Analysis Output', 'Ideogram Annotations']
+                      'Other', 'Analysis Output', 'Ideogram Annotations', 'Image']
```

Comment:
Thinking out loud - should this be a more specific label than "Image"?  If we get different image types later, we may want that (but maybe we also want the generality here - not really sure).

Reply:
Yeah, it's a good question to keep in the back of our minds as we're doing this.  I suspect that we'll have a clear answer to this the first time we want to create a study_file_bundle with an image